tests/conftest.py

```diff
@@ -261,21 +261,36 @@ def kubeconfig_export_path():
 def exported_kubeconfig(unprivileged_secret, kubeconfig_export_path):
     if not unprivileged_secret:
         yield
+
     else:
-        kubeconfig_path = tempfile.mkdtemp(suffix="-cnv-tests-kubeconfig")
-        LOGGER.info(f"Kubeconfig for this run is: {kubeconfig_path}")
-        if not os.path.isdir(kubeconfig_path):
-            os.mkdir(kubeconfig_path)
+        kube_config_path = os.path.join(os.path.expanduser("~"), ".kube/config")
```

Comment:
maybe switch to pathlib

Reply:
in some places we need str (e.g when returning the path for get_client) and when setting kubeconfig, I think it is clearer to use only str and not both Path and str.